mathquill4quill.js

```diff
@@ -357,6 +395,7 @@ window.mathquill4quill = function(dependencies) {
     let historyList = fetchHistoryList(historyCacheKey);
     const historySize = options.historySize || 10;
     const displayHistory = options.displayHistory || false;
+    const displayDeleteButtonOnHistory = options.displayDeleteButtonOnHistory || false;
```

Comment:
Nit: apply `./node_modules/.bin/prettier --write "**/*.js"` so that CI passes.
```suggestion
    const displayDeleteButtonOnHistory =
      options.displayDeleteButtonOnHistory || false;
```